README.md

```diff
@@ -1,6 +1,40 @@
 # Quedex Python API
 
-## Prerequisites
+A library used to communicate with [Quedex Bitcoin Derivatives Exchange](https://quedex.net). If you 
+have any questions please contact support@quedex.net or open an issue in this repository.
 
-The following packages need to be installed on system level:
-* libffi-dev
+## Using the API
+
+A simple example of how to use the API is available in 
+[Simple Trading Tutorial](docs/tutorials/simple_trading.py.md) - it shows how to set it up using
+[Autobahn](http://crossbar.io/autobahn/) WebSocket implementation on top of 
+[Twisted](https://www.twistedmatrix.com/). The tutorial is written in literate programming style and
```

Comment:
I'd move the remark on literate programming to some 'Why are we cool?' paragraph at the end :sunglasses: